main.py

```diff
@@ -58,7 +58,7 @@ def upload_file(
     return response
 
 
-@app.get("/download_bee/")
+@app.get("/download_bee/", status_code=204)
```

Comment:
This seems a delete status code: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204